buildroot-external/board/hardkernel/odroid-xu4/partition-spl-spl.cfg

```diff
@@ -0,0 +1,6 @@
+partition spl {
+	size = ${BOOT_SPL_SIZE}
+	image = "spl.bin"
+	in-partition-table = "no"
+	offset = 32K
```

Comment:
Absolutely, I also see that XU4 and M1/M1S (where I forgot to stage SPL configs) failed in my test build, so this is probably the cause.

Reply:
Ahh, I see what's wrong, the build failed because I have `spl.bin` here. It should be `spl.img`, but since I ran a non-clean build, I still had the other file from previous tests and didn't notice.

But the hole is not necessary, because 0-32k part of the image is untouched, this SPL is written to the 32k offset.